sdk/component/store/remote/postgresql/sqlc/migrations/20241209103854_findings_table.sql

```diff
@@ -0,0 +1,8 @@
+-- create "findings" table
+CREATE TABLE IF NOT EXISTS finding (
+   id SERIAL PRIMARY KEY,
+   instance_id UUID NOT NULL UNIQUE,
```

Comment:
If this is at a finding level, instance_id can not be unique

Reply:
Good spot

Reply:
Fixed.